src/backend/metal/src/command.rs

```diff
@@ -37,6 +37,14 @@ const CLEAR_IMAGE_ARRAY: bool = false;
 /// Number of frames to average when reporting the performance counters.
 const COUNTERS_REPORT_WINDOW: usize = 0;
 
+//TODO: use for inserting points of interest into System Trace
```

Comment:
This seems useful! 👀

Reply:
an important caveat that those only show up in the "System Trace", *not* "Metal System Trace".

Reply:
note: this PR is getting dropped, so we need to keep an eye for those extern interfaces and bring them in separately